test/bunyan.test.js

```diff
@@ -268,6 +268,31 @@ describe('bunyan-logger', function() {
           done();
         });
   });
+
+  it('test options.levelFn', function (done) {
+    var app = express();
+    var output = st();
+    app.use(bunyanLogger({
+      stream: output,
+      levelFn: function (status, err, meta) {
```

Comment:
That api is a bit strange.
Later we can probably move that to the first argument as `{status, meta}`.
But that means we have to make a major version bump.